.github/workflows/41-railway-setup.yml

```
@@ -53,82 +41,45 @@ concurrency:
+      # the app images to this run's tag, wait for green, and smoke it — all in
+      # one script, pure GraphQL (no Railway CLI). The deploy workflow (43)
+      # then only re-verifies and posts the preview URL.
+      - name: Create or update preview environment
         id: clone
         env:
           PR_NUMBER: ${{ inputs.pr_number }}
           IMAGE_TAG: ${{ inputs.image_tag }}
+          # The acceptance suites authenticate admin calls with this key;
+          # clones inherit the template's own generated key, so the script
+          # upserts the CI key onto the clone before anything deploys.
+          AGENTA_AUTH_KEY: ${{ secrets.AGENTA_TEST_OSS_AUTH_KEY }}
```

actionlint (1.7.12)

[error] property "agenta_test_oss_auth_key" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; github_token: string; railway_token: string}

(expression)